### ISSUE TITLE
About save image function

### ISSUE THREAD

**Issue by SMY19999** [luke23571113/nvdiffrast-testing#1]
Hi Luke!

I'm studying nvdiffmodeling. Thanks for your testing code repository!

The function `save_image` in `main.py`  only has a small bug.

Just change ` img = img.detach().cpu().numpy()` to `img = img.detach().cpu().numpy()[0]`, this will work.

It is because, in deep learning, the shape of data can usually be (batch, M, N).
